module-2/backend/week_one.md

```diff
@@ -1,32 +1,83 @@
 ## Week One - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!).
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+> get, displays view
+  redirect, calls for different uri
+  post, create new data/object
+  put, update data/object
+  delete, delete data/object
+
 2. What is Sinatra?
+> Sinatra is a framework that enables the use of views and controllers which accepts and reads HTTP requests.
+
 4. What is MVC?
+> Model, view, and controller.
+  This is the set up that allows for the processing HTTP requests.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+> These conventions follow the CRUD approach and allows for a consistent approach to the use of Sinatra methods.
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+> instance variables
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
-  
+
   ```ruby
   get '/horses' do
+  **@count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+  @name = 'Mr. Ed' in line 31

```

Comment:
This is still an instance variable. In order to pass locals we need a little bit more work:

```ruby
   get '/horses' do
      @count = 1
      name = "Mr. Ed"
      erb :index, locals: {name: name}
    end
```